ckanext/tracking/cli/tracking.py

```diff
@@ -53,8 +53,7 @@ def update_all(start_date: Optional[str] = None):
         # No date given. See when we last have data for and get data
         # from 2 days before then in case new data is available.
         # If no date here then use 2011-01-01 as the start date
-        stmt = select(ts).order_by(ts.tracking_date.desc())
-        result = session.scalars(stmt).first()
+        result = session.query(ts).order_by(desc(ts.tracking_date)).first()
```

Comment:
👍 

Since we throw away the scalars list of objects there no point in pulling it into memory when we only care about the first record. 

I'd say there is heaps of things like this since its very easy to mess up in ORM systems when it works with tiny table sizes. It should also help in reducing Postgres cpu/disk scan's getting all records which are just thrown on the floor.